src/main/java/com/google/api/codegen/ruby/RubyGapicContext.java

```diff
@@ -330,6 +330,10 @@ public String rubyTypeName(TypeRef typeRef) {
     return modules;
   }
 
+  public Iterable<String> getApiModules() {
+    return Splitter.on("::").splitToList(getApiConfig().getPackageName());
```

Comment:
Optional: This is sort of duplicated code -- it has the same functionality as the `pathMapper` that's passed into the Ruby `CommonGapicProvider`. I'm not sure it's worth going through the contortion of eliminating this duplicated code. But potentially you could pass the `pathMapper` into `RubyGapicContext` to get ride of the duplication.